plugin/completion.py

```diff
@@ -1,36 +1,83 @@
 import sublime
 import sublime_plugin
-from .core.completion import parse_completion_response, format_completion
-from .core.configurations import is_supported_syntax
-from .core.documents import position_is_word
-from .core.edit import parse_text_edit
+
+from .core.protocol import Request, Range, InsertTextFormat
```

Comment:
I've tried this branch with `LSP-json` as completions have issues in old code.

With new code it's better, I don't get duplicate quotes when completing values in `*.sublime-settings` file. For example in:
```js
	"default_line_ending": "unix",
```
Changing "unix" to something else with help of completions insert correct amount of quotes.
But I found another problem:
1. Select `unix` value and delete it
2. Trigger completions and pick some completion

Problem: The value and quotes are inserted correctly but trailing comma gets removed.

Reply:
I am aware of that issue. 🙂 I know the cause but haven't found a solid solution.